internal/database/postgres/pg.go

```diff
@@ -82,7 +82,11 @@ func (c *Config) Connect(useAdmin bool, pusherRatio, spoolerRatio float64, purpo
 	if err != nil {
 		return nil, err
 	}
-	config.MaxConns = int32(connConfig.MaxOpenConns)
+
+	if connConfig.MaxOpenConns != 0 {
```

Comment:
Do you want to set a default value for `config.MaxConns` in case it is 0?

Reply:
The default is set by `pgxpool` itself, tl;dr: `max(<num_cpus>, 4)` which seems alright

Reply:
aha got it. THanks